src/components/progress/progress.scss

```diff
@@ -0,0 +1,114 @@
+@import '../../themes/ionic.globals';
+// Progress bar
+// --------------------------------------------------
+
+/// @prop - The color of the element that is being animated
+$progress-color:rgb(16,108,200) !default;
+
+/// @prop - The background color of the progress bar
+$progress-buffer-color:  rgb(170,209,249) !default;
+
+/// @prop - The animation duration of the progress bar
+$progress-animation-duration: 2s !default;
+
+/// @prop - The speed at which the progress values transition, this is useful for determinate and buffer modes
+$progress-values-transition: 0.3s !default;
+
+/// @prop - The speed at which the buffer circles will be animating
+$progress-buffer-circle-speed: 0.3s !default;
+
+ion-progress {
+    height: 5px;
+    width: 100%;
+    position: relative;
+    overflow: hidden;
+    display: block;
+    @keyframes indeterminate {
+        0% {
+            width: 0%;
+
+            left: 0%;
```

Comment:
Instead of `left`, use:
```
@include multi-dir() {
  // scss-lint:disable PropertySpelling
  left: 0%;
}
```

same for all keyframes.

*It is needed for if any class wants to override it